core/agents/spec_writer.py

```diff
@@ -75,6 +83,27 @@ async def run(self) -> AgentResponse:
         self.next_state.action = SPEC_STEP_NAME
         return AgentResponse.done(self)
 
+    async def update_spec(self) -> AgentResponse:
+        feature_description = self.current_state.current_iteration["description"]
```

Comment:
I understand that in this case it doesn't make too much difference but I would prefer using  `self.current_state.current_iteration["user_feedback"]` for code readability/understanding